packages/@aws-cdk/toolkit-lib/lib/actions/bootstrap/index.ts

```diff
@@ -169,6 +169,25 @@ export interface BootstrapParameters {
   readonly customPermissionsBoundary?: string;
 }
 
+export interface EnvironmentBootstrapResult {
+  environment: string;
```

Comment:
Would this be better types as ?
```suggestion
  environment: cxapi.Environment;
```

Reply:
Replaced and updated tests as well.